src/journal/journald-server.c

```diff
@@ -853,7 +853,10 @@ static void write_to_journal(Server *s, uid_t uid, struct iovec *iovec, size_t n
                 return;
         }
 
-        log_info_errno(r, "Failed to write entry (%zu items, %zu bytes), rotating before retrying: %m", n, IOVEC_TOTAL_SIZE(iovec, n));
+        if (r == -E2BIG)
+                log_info("Journal file %s is full, rotating to a new file", f->file->path);
```

Comment:
Maybe, `log_debug()` is enough in this case?

Reply:
Having a log message when a file was rotated is quite useful. Unless an extreme amount of data is being logged, this message shouldn't trigger all that often so I don't think it hurts to keep it at info.

Reply:
Nvm, this could actually be triggered very often depending on what file size limits are configured. Downgraded to log_debug()